app/controllers/users/confirmations_controller.rb

```diff
@@ -2,8 +2,12 @@ module Users
   class ConfirmationsController < Devise::ConfirmationsController
     include ValidEmailParameter
 
+    def new
+      @user = User.new
```

Comment:
another reason I don't love `resource` as a name is that it is so general. I understand why the devise lib wants to be general, but for the purposes of our app I think `@user` is clearer